CHANGELOG.md

```diff
@@ -5,5 +5,5 @@ Bob Management GUI changelog
 ## [Unreleased]
 
 #### Added
-- Initial project structure, backend only (#1)
 
+- Initial project structure (#1)
```

Comment:
Please, add a separate line for this request. You can write the number of PR inside braces